internal/graph/check_test.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/openfga/openfga/pkg/logger"
 	"github.com/openfga/openfga/pkg/storage"
 	"github.com/openfga/openfga/pkg/storage/memory"
+	"github.com/openfga/openfga/pkg/storage/storagewrappers"
```

Comment:
@jpadilla going back to https://github.com/openfga/openfga/pull/2117#discussion_r1862797277

I would need to move your storage wrapper together with the other storage wrappers, otherwise the linter complains:

```
internal/graph/builder.go:1: : import cycle not allowed in test (typecheck)
package graph
```